extension-manifest-v3/src/background/utils/setup.js

```diff
@@ -14,7 +14,7 @@ export default function asyncSetup(promises, threshold = 5000) {
 
   const result = {
     pending: Promise.race([
-      Promise.all(promises),
```

Comment:
why is this needed?

Reply:
I was in one of the implementation when I was passing ternary in `adblocker.js` - but it is not now, I'll clean it up.